virt/release_notes/virt-4-19-release-notes.adoc

```diff
@@ -92,17 +92,14 @@ As a cluster administrator, you can prevent users from enabling VM delete protec
 
 [id="virt-4-19-storage_{context}"]
 === Storage
+// CNV-52871 - Doc: Multi IOthread with fast storage
```

Comment:
Is the comment needed?

I don't use screen readers but they might not pick up the inline link. Consider adopting the "For more information, see" link clause. 

The link title does not match the target section title: Configuring multiple IOThreads for fast storage access